### PR TITLE
Update trial documentation

### PR DESCRIPTION
**Pull request by alorenzo175**
mainly making times more precise and more warning that the script is only an example

### CODE REVIEW

**Comment by alorenzo175** [comment]
separately added the prob. forecast upload to the gist of the example script

File: documentation/dashboard/trials.md [hunk 1]
```diff
@@ -297,7 +301,7 @@ the solarforecastarbiter-core Python package can be installed from the
 or via pip with the command
 
 ```
-pip install git+https://github.com/solararbiter/solarforecastarbiter-core.git
+pip install solarforecastarbiter
```
Comment (wholmgren) [inline documentation/dashboard/trials.md:304]: needs `--pre`?

**Review by wholmgren — APPROVED** [review]

File: documentation/dashboard/trials.md (hunk 1 above)
Comment (alorenzo175) [reply documentation/dashboard/trials.md:304]: didnt' for me. can you test too?

Comment (wholmgren) [reply documentation/dashboard/trials.md:304]: indeed. https://www.python.org/dev/peps/pep-0440/#handling-of-pre-releases

> Pre-releases of any kind, including developmental releases, are implicitly excluded from all version specifiers, unless they are already present on the system, explicitly requested by the user, or if the only available version that satisfies the version specifier is a pre-release.

**Pull request merged**